draft-ietf-ppm-dap.md

```diff
@@ -2121,6 +2128,54 @@ send an HTTP DELETE request to
 `{helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}` so that the
 Helper knows it can clean up its state.
 
+#### Batch Buckets {#batch-buckets}
+
+When aggregation recovers an output share, it must be stored into an appropriate
+"batch bucket", which is defined in this section. The data stored in a batch
+bucket is kept for eventual use in the {{collect-flow}}.
+
+Batch buckets are indexed by an aggregation parameter, as well as a batch
+mode-specific "batch bucket identifier":
+
+* For the time-interval batch mode, the batch bucket identifier is an interval
+  of time whose beginning is a multiple of the task's `time_precision` and whose
+  duration is equal to the task's `time_precision`. The identifier associated
+  with a given output share is the unique such interval containing the client
+  timestamp of the report the output share was recovered from.
+* For the leader-selected batch mode, the batch bucket identifier is a batch ID.
+  The identifier associated with a given output share is the batch ID of the
+  related aggregation.
+
+A few different pieces of information are associated with each batch bucket:
+
+* An VDAF-specific aggregate share, as defined in {{!VDAF}}.
+* A count of a number of reports included in the batch bucket.
+* A 32-byte checksum value.
+
+When processing an output share `out_share`, the following procedure is used to
+update the batch buckets:
+
+* Look up the existing batch bucket for the aggregation parameter `agg_param`
+  associated with the aggregation job, and the batch bucket identifier
+  associated with the aggregation job and output share.
+  * If there is no existing batch bucket, initialize a new one. The initial
+    aggregate share value is computed as `Vdaf.agg_init(agg_param)`; the initial
```

Comment:
Add reference to section in the VDAF draft that will define this. Draft 13 isn't out yet, but the section number is stable at this point (https://cfrg.github.io/draft-irtf-cfrg-vdaf/draft-irtf-cfrg-vdaf.html#section-5.4).